rsqueakvm/model/compiled_methods.py

```diff
@@ -184,8 +191,15 @@ def getliteral(self, index):
     def primitive(self):
         return self._primitive
 
+    def frame_size(self):
+        return self._frame_size
+
+    def update_frame_size(self, size):
+        if self._frame_size < size:
+            self._frame_size = size
+
     @jit.elidable_promote()
-    def compute_frame_size(self):
+    def squeak_frame_size(self):
```

Comment:
We now report the frame size as viewed by Squeak differently to the one that we discovered while running (stored in `_frame_size`).